packages/components/src/components/modal/_modal.scss

```diff
@@ -75,6 +75,8 @@
     width: 100%;
     height: 100%;
     max-height: 100%;
+    outline: 3px solid transparent;
```

Comment:
Could you add a quick note to explain the outline additions? Make a comment plus link to issue or something?